tests/functional/adapter/test_on_schema_change.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+import pytest
+
+from dbt.contracts.results import RunStatus
+from dbt.tests.util import run_dbt, run_dbt_and_capture
+
+models__table_base_model = """
+{{
+  config(
+    materialized='incremental',
+    incremental_strategy='append',
+    on_schema_change=var("on_schema_change"),
+    table_type=var("table_type"),
+  )
+}}
+
+select
+    1 as id,
+    'test 1' as name
+{%- if is_incremental() -%},
+    current_date as updated_at
+{%- endif -%}
```

Comment:
```suggestion
{%- if is_incremental() -%}
    ,current_date as updated_at
{%- endif -%}
```

Reply:
SQL gets really ugly when you add preprocessors to it… I was aiming for nice SQL, but you're right that more readable code is higher priority.